READMEs/marketplace-flow.md

```diff
@@ -101,6 +101,7 @@ In the work console:
 #set private keys of two accounts
 export TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
 export TEST_PRIVATE_KEY2=0x804365e293b9fab9bd11bddd39082396d56d30779efbb3ffb0a6089027902c4a
+export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58
```

Comment:
Why is this added? Is it deploying a new factory? It shouldn't be - it should be using the contracts already deployed, with addresses specified in addresses.json.

Reply:
I saw that the ganache wallet uses this key, so that's why I have exported it before. Should I replace it?

Reply:
I can't comment more on this PR until I see a github issue reporting what this PR is trying to fix.

Reply:
Thanks for creating the github issue, and linking to it here. +1

It doesn't seem logical that someone using a quickstart should have to point to the private key of some deployer. The only private keys a quickstart should really have are those of the user. So, something's up. You'll need to get to the bottom of this.

Reply:
I thought more about it. This is really here in order to mint fake ocean. Alright, fair enough.

But you need to make it more clear why that private key is being exported. For example, this part of the README could be like:
```console
#set private keys of two accounts
export TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
export TEST_PRIVATE_KEY2=0x804365e293b9fab9bd11bddd39082396d56d30779efbb3ffb0a6089027902c4a

#needed to mint fake OCEAN
export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58

#start python
python
```

Reply:
Ok, will modify right now.